src/pages/[platform]/build-a-backend/storage/download-files/index.mdx

```diff
@@ -1116,6 +1123,7 @@ final operation = Amplify.Storage.downloadFile(
       getProperties: true,
       useAccelerateEndpoint: true,
     ),
+    bucket: StorageBucket.fromOutputs('secondBucket'),
```

Comment:
Q: This is optional right? Would a more explicit example and/or description be beneficial?

Reply:
I will coalesce them into their own section about multi bucket like the other apis